docs/maui/TOC.yml

```
@@ -183,3 +183,5 @@ items:
       href: markup/extensions/style.md
     - name: "VisualElement extensions"
       href: markup/extensions/visual-element-extensions.md
+    - name: "View extensions"
+      href: markup/extensions/view-extensions.md
```

Minor alphabetical ordering fix

```suggestion
    - name: "View extensions"
      href: markup/extensions/view-extensions.md
    - name: "VisualElement extensions"
      href: markup/extensions/visual-element-extensions.md
```